packages/legend-application-studio/src/components/__tests__/EmbeddedQueryBuilder.test.tsx

```diff
@@ -253,6 +429,175 @@ test(integrationTest('Open query builder by querying a class'), async () => {
   );
 });
 
+test(
+  integrationTest(
+    'Open query builder by querying a processing temporal milestoned class',
+  ),
```

Comment:
Do we have a test for the most recent bug we found where date propagation is not supported for groupBy Functions. 
In this case we are expecting query builder not to force propagate the date. 
We should also have tests when this behavior is expected. 
Right now if the user opens a query that has dates defined but are expected to be propagated we rewrite this, we should add tests for those on the expected input vs output.

Reply:
yeah i think expected input vs output tests would make much sense for milestoning. Thanks for the idea